File: Backend/Src/Models/ChatMessage.cs
namespace EmergencyDepartment.Models;

public partial class ChatMessage
{
	public int Id { get; set; }

	public string? Text { get; set; }

	public string? ImageUrl { get; set; }

	public DateTime ReceivedDate { get; set; }

	public long SenderPersonNumber { get; set; }

	public virtual User? Sender { get; set; }

	public long ReceiverPersonNumber { get; set; }

	public virtual User? Receiver { get; set; }

	public int InquiryId { get; set; }

	public virtual Inquiry? Inquiry { get; set; }
}

File: Frontend/src/models/ChatMessage.ts
import { Inquiry } from "./Inquiry";
import { User } from "./User";

export interface ChatMessage {
	id: number;
	text?: string;
	imageUrl?: string;
	receivedDate?: string;
	senderPersonNumber: number;
	sender?: User;
	receiverPersonNumber: number;
	receiver?: User;
	inquiryId: number;
	inquiry?: Inquiry;
}
